com.microsoft.mrtk.uxcore/Keyboard/NonNative/NonNativeKey.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.UX
+{
+    /// <summary>
+    /// Abstract class representing a key in the non native keyboard
+    /// </summary>
+    public abstract class NonNativeKey : MonoBehaviour
+    {
+        /// <summary>
+        /// Reference to the GameObject's button component.
+        /// </summary>
+        [field: SerializeField, Tooltip("Reference to the GameObject's button component.")]
```

Comment:
WHOA I have not seen this syntax before. Does this do what I think it does...???

Reply:
i.e., does this synthesize a `.button` field behind the autoproperty?

Reply:
It serializes a PROPERTY! Yeah!

Reply:
Your comment is more accurate, it's serializing the backing field.